docs/faq.md

```diff
@@ -83,8 +83,39 @@ We list some common troubles faced by many users and their corresponding solutio
     1. This error indicates that your module has parameters that were not used in producing loss. This phenomenon may be caused by running different branches in your code in DDP mode.
     2. You can set ` find_unused_parameters = True` in the config to solve the above problems or find those unused parameters manually.
 
+- Save the best model
+
+    It can be turned on by configuring `evaluation = dict(save_best=‘auto’)`. In the case of the `auto` parameter, the first key in the returned evaluation result will be used as the basis for selecting the best model. You can also directly set the key in the evaluation result to manually set it, for example, `evaluation = dict(save_best='mAP' )`.
+
+- Usage of EMA Hook in Resume
+
+    If you use `EMA Hook` in training, you can't use command line parameters such as `--resume-from` and `--cfg-options resume_from`to restore model parameters during resume, you can only do so by modifying the `resume_from` field in the configuration file. This is because EMA Hook cannot obtain command line parameters, and thus cannot implement the weight reload function.
```

Comment:
May give an example, still feel confused about the `resume_from` field in the configuration file`